### DISCUSSION TITLE
Training hybrid deepmd - LJ/Buckingham

### DISCUSSION THREAD

**Discussion by hoba87** [deepmodeling/deepmd-kit#2598] [Q&A]
I am seeing this question #2395 and I am wondering if it is possible to train a hybrid deepmd model which is combined with a LJ or Buckingham model and if yes how to do it?

**Comment by njzjz** [comment]
In the case #2395, a correction model should be trained that corrects the energy from L-J to DFT.

```math
E^*=E_{\text{DFT}}-E_{\text{LJ}}
```

**Reply by hoba87** [reply]
Is it possible to train the correction model with deepmd? I have substracted it as you proposed, however the training seems to fail completely (loss is at values of 10^4, 10^5)?

**Reply by njzjz** [reply]
Please first confirm you also substracted the forces.

The L-J interaction is quite large and inaccurate for some structures. These structures should not be added to the training set.

**Reply by hoba87** [reply]
The forces and the virial are also substracted. Ok I have to check if there are large deviation for some structures.

**Reply by hoba87** [reply]
Also with removing the structures with largest deviation, the loss looks the same for me.

**Reply by njzjz** [reply]
Do you use the correct unit? What is the largest L-J energy and force?

**Reply by hoba87** [reply]
Ah, ok I have used the incorrect unit for the virial (bar instead of eV). The energy is shifted by ~3000eV and forces are about one magnitude larger than DFT, but should be correct units.
Not sure if the training is successful (still running), however at least the loss is in the range of 10^0 - 10^1 now.

**Reply by hoba87** [reply]
In principle it is working now, however the results are not very good. Thank you for support.
